src/store/modules/TsModsModule.ts

```diff
@@ -192,6 +176,28 @@ export const TsModsModule = {
         },
         updateDeprecated(state, allMods: ThunderstoreMod[]) {
             state.deprecated = Deprecations.getDeprecatedPackageMap(allMods);
+        },
+        prewarmCacheMod(state: State, mods: ThunderstoreMod[]) {
```

Comment:
I made an earlier comment about this method "no longer" being about prewarming the cache, but I confused it with the old `prewarmCache` below. Still, this is more about updating the cache so another name might be better?